src/server/app.ts

```diff
@@ -14,7 +14,7 @@ const renderer = createBundleRenderer(
 	{ runInNewContext: false },
 );
 
-app.get( '/termbox', ( request, response ) => {
+app.get( '/termbox', ( request: Request, response: Response ) => {
```

Comment:
Hm, my IDE seems to guess the correct type, but then still gives different hints once the type is made explicit. 

![req](https://user-images.githubusercontent.com/453024/50147447-74e55200-02b6-11e9-9949-d0406809358c.png)

Reply:
I won a XMLHttpRequest which was less than helpful.